packages/vinegar-git/vinegar-git.pacscript

```diff
@@ -0,0 +1,49 @@
+pkgname="vinegar-git"
+repology=("project: vinegar")
+arch=('amd64')
+pkgver="1.7.8"
+url='https://vinegarhq.org/'
+source=("https://github.com/vinegarhq/vinegar.git")
+makedepends=(
+  "golang"
```

Comment:
```suggestion
  "golang>=2:1.23"
```

https://github.com/pacstall/pacstall-programs/actions/runs/12538206571/job/34963102165?pr=6781#step:11:1082

Reply:
I opted for `>=2:1.22` since upstream specifies it at https://github.com/vinegarhq/vinegar/blob/master/go.mod#L3.